src/backend/dx12/src/device.rs

```diff
@@ -1382,6 +1420,11 @@ impl d::Device<B> for Device {
             (rtvs, num_rtvs)
         };
 
+        let sample_desc = dxgitype::DXGI_SAMPLE_DESC {
+            Count: desc.multisampling.as_ref().map_or(1, |ms| ms.rasterization_samples as _),
```

Comment:
nit: at some point, the chain of commands with closures becomes less readable than a simple `match`